res/controllers/common-controller-scripts.js

```diff
@@ -421,10 +421,18 @@ bpm.tapButton = function(deck) {
     var now = new Date() / 1000   // Current time in seconds
     var tapDelta = now - bpm.tapTime
     bpm.tapTime = now
+    if (bpm.tap.length < 1.0) {
```

Comment:
Isn't `bpm.tap.length` and intenger?
```suggestion
    if (bpm.tap.length < 1) {
```